docs/Enable-NTFSAccessInheritance.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Enable-NTFSAccessInheritance
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Enables inheritance of Access Control Entries from the target directory's parent.